src/shared/components/ShareWithColleagues/ShareWithColleagues.helpers.tsx

```diff
@@ -1,10 +1,16 @@
-import { flatten, groupBy } from 'lodash';
+import { flatten, groupBy, isEmpty, remove } from 'lodash';
 
 import { ShareUserInfo, ShareUserInfoRights } from './ShareWithColleagues.types';
 
 export const sortShareUsers = (users: ShareUserInfo[]) => {
 	const groupedUsers = groupBy(users, 'rights');
-	return flatten([groupedUsers.EIGENAAR, groupedUsers.BIJDRAGER, groupedUsers.KIJKER]);
+
+	return flatten(
+		remove(
+			[groupedUsers.EIGENAAR, groupedUsers.BIJDRAGER, groupedUsers.KIJKER],
+			(group) => !isEmpty(group)
+		)
+	);
```

Comment:
descontructing an empty array with the spread operator (...) doesn't create any elements, so this should work:
```suggestion
	return [...groupedUsers.EIGENAAR, ...groupedUsers.BIJDRAGER, ...groupedUsers.KIJKER];
```